src/lib/kit/constants/config.tsx

```diff
@@ -105,6 +105,7 @@ export const dynamicConfig: DynamicFormConfig = {
             row: Row,
             row_verbose: RowVerbose,
             table_item: TableCell,
+            transparent: Transparent,
```

Comment:
you also need to add a transparent layout for the dynamicViewConfig for boolean spec

Reply:
fixed.

@NasgulNexus plz, look at https://preview.gravity-ui.com/dynamic-forms/171/?path=/story/object-inline--inline
I've added checkbox. Looks like `true|false` on the `View` tab.

Perhaps it would be cool to set aliases in spec for the checkbox values? But I'm not sure that anyone needs it.